apps/widget-player/src/styles.css

```
@@ -0,0 +1,1404 @@
+:root {
+  color: #34322e;
+  background: #f7f5f2;
+  font-family:
+    ui-rounded, "SF Pro Rounded", "Avenir Next", "Segoe UI", system-ui,
+    sans-serif;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)